pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -286,6 +286,10 @@ func (t *EventListenerTrigger) validate(ctx context.Context) (errs *apis.FieldEr
 
 	// Validate optional Interceptors
 	for i, interceptor := range t.Interceptors {
+		// No continuation if provided interceptor is nil.
+		if interceptor == nil {
+			return errs.Also(apis.ErrInvalidValue(fmt.Sprintf("interceptor '%v' must be a valid value", interceptor), fmt.Sprintf("interceptors[%d]", i)))
```

Comment:
Could we add a quick unit test for this?

Reply:
@dibyom added unit test 
PTAL